OracleAnalytics/patches/6.4-patch/README.md

```diff
@@ -0,0 +1,34 @@
+# Build a Oracle Analytics Server 6.4 Patched Image
+
+This Dockerfile extends the Oracle Analytics Server image (6.4),
+and applies necessary patches.
+
+## How to build and run
+
+First make sure you have built `oracle/analyticsserver:6.4`.
+
+Download the following patches from
+[My Oracle Support](https://support.oracle.com)
+and place them in the same directory as this README.

```

Comment:
You need to make it clear that this option is only available to customers who have the appropriate paid licenses and thus access to My Oracle Support.